src/js/css.js

```diff
@@ -1,14 +1,10 @@
-import { isBrowserSupported } from './browser'
-
 /**
  * Injects a string of CSS styles to a style node in <head>
  * @param {String} css
  */
 export function injectCSS(css) {
-  if (isBrowserSupported) {
```

Comment:
I think without this check it will break SSR when using `index.all.js`, because `document is not supported` error.